tests/test_api_spanish.py

```
@@ -6,7 +6,20 @@
 
-from app import app
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+# Import the FastAPI app from app.py file
+import importlib.util
+
+spec = importlib.util.spec_from_file_location("app_module", "app.py")
+if spec is None or spec.loader is None:
+    raise ImportError("Cannot load app.py")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)